packages/holocron/src/index.js

```diff
@@ -69,4 +75,7 @@ export {
   setRequiredExternalsRegistry,
   clearModulesUsingExternals,
   getModulesUsingExternals,
+  getStoredPromise,
+  storePromise,
+  useStoredPromise,
```

Comment:
Expose only the helper functions, to maintain abstraction for future underlying changes